### ISSUE TITLE
Update calibration docs to reduce confusion regarding builtin vs CalibDB

### ISSUE THREAD

**Issue by DeltaDizzy** [PhotonVision/photonvision#1361]

**Comment by mcm001** [comment]
Yeah we should clarify that calibdb is good for practice, but mrcal is better in every other way

**Comment by Gold856** [comment]
#1396 removed CalibDB and there seems to have been a docs change made separately so that CalibDB is just not even mentioned. Closing as fixed.
